app/views/hosts/_templates.html.erb

```diff
@@ -0,0 +1,25 @@
+<% if @templates.is_a? Array %>
+  <table class="table table-bordered table-striped">
+    <thead>
+      <tr>
+        <th><%= _("Template Type") %></th>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @templates.each do |tmpl| %>
+        <tr>
+          <td><%= (_("%s Template") % tmpl.template_kind.to_label) %></td>
```

Comment:
Why not move the `" Template"` part into the jar as well? that way you can make sure the translators get the correct string to translate, e.g. "Provision template" instead of translating "Provision" and "Template" separately, which may have a different translation.